Chandra/cmd_states/cmd_states.py

```diff
@@ -382,9 +382,14 @@ def get_state0(date=None, db=None, date_margin=10, datepar='datestop'):
      time.  The conservative default value of ``date_margin=10`` allows for
      processing downtime.  The table is accessed via the ``db`` object.
 
+     If ``date_margin`` is None then no date margin is applied and the most
+     recent available state that meets the date selection criteria (including
+     that pcad_mode == 'NPNT') will be used.  In this case a ``date`` must be
```

Comment:
If `date is None` and `date_margin is None` then it calls the following with `date=None`, which is wrong:
```
state0 = db.fetchone("""SELECT * FROM cmd_states
                              WHERE %s < '%s'
                              AND pcad_mode = 'NPNT'
                             ORDER BY %s DESC""" % (datepar, date, datepar))
```